src/foundry/common/data/data.mjs/journalEntryData.d.ts

```diff
@@ -1,16 +1,16 @@
 import { ConfiguredDocumentClass, ConfiguredFlags, PropertiesToSource } from "../../../../types/helperTypes";
+import type EmbeddedCollection from "../../abstract/embedded-collection.mjs";
```

Comment:
This entire file touches too closely to a bunch of `DataModel` stuff updated elsewhere. This'll make it a pain to merge this PR or another. I would just revert these changes. I understand it makes things inaccurate but this update will be handled, just elsewhere.

Sorry for your sunk work in this file, `DataModel` just effected document stuff too intimately.

Reply:
hm. any recs on what to do in the mean time if i want to use the types in system/module development?

Reply:
also: is there any place i should be checking to ensure i'm not doing redundant work in the future? i didn't spot anything in the open issues or PRs that suggested it was underway (or otherwise to be avoided), but maybe i missed something 🤔